src/containers/ProductDetails/ProductDetails.tsx

```diff
@@ -41,22 +39,26 @@ export const ProductDetails = ({
   };
 
   const handleAddToCartPress = () => {
-    navigation?.navigate(modalRoutes.login);
+    navigation?.navigate(modalRoutes.loginRequired);
   };
 
   useEffect(() => {
     getProductDetails();
   }, []);
 
   if (isLoading) {
-    return <Typography>Loading ...</Typography>;
+    return (
+      <SafeAreaView>
+        <Typography>Loading ...</Typography>
+      </SafeAreaView>
+    );
   }
 
   return (
     <>
-      <ScrollView stickyHeaderHiddenOnScroll>
-        <Header type="product" navigation={navigation} />
-      </ScrollView>
+      <SafeAreaView>
+        <Header />
```

Comment:
Add header as Stack navigator props, don't use it directly in each screen

Reply:
done